### PR TITLE
tests: fix eval dataset

### PR DESCRIPTION
**Pull request by MartinBernstorff**
# Notes for reviewers
Synthetic dataset generation is now in `tests/test_data/model_eval/generate_synthetic_dataset_for_eval.py`

I'm not sure why you got so many event timestamps that were identical to the prediction timestamp. Let me know if this is still the case - or feel free to modify the code in `generate_synthetic_dataset_for_eval` :-)

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -14,19 +14,6 @@
...
 
 
-def add_age_is_female(df: pd.DataFrame):
-    """Add age and gender columns to dataframe.
-
-    Args:
-        df (pd.DataFrame): The dataframe to add age and sex cols to.
-    """
-    ids = pd.DataFrame({"dw_ek_borger": df["dw_ek_borger"].unique()})
-    ids["age"] = np.random.randint(17, 95, len(ids))
```
Comment (sarakolding) [inline tests/conftest.py:24]: 17? Thought we looked at 18+ only

File: tests/conftest.py [hunk 2]
```diff
@@ -14,19 +14,6 @@
...
-    ids["is_female"] = np.where(ids["dw_ek_borger"] > 30_000, 1, 0)
```
Comment (sarakolding) [inline tests/conftest.py:25]: Everyone with id over 30,000 are women? bit confused 🤔

File: tests/conftest.py [hunk 3]
```diff
@@ -41,15 +28,15 @@ def add_eval_column(df: pd.DataFrame) -> pd.DataFrame:
 @pytest.fixture(scope="function")
 def synth_eval_dataset() -> EvalDataset:
     """Load synthetic data."""
-    csv_path = Path("tests") / "test_data" / "synth_eval_data.csv"
+    csv_path = Path("tests") / "test_data" / "model_eval" / "synth_eval_data.csv"
     df = pd.read_csv(csv_path)
-    df = add_eval_column(df)
-    df = add_age_is_female(df)
```
Comment (sarakolding) [inline tests/conftest.py:47]: no age and sex?

File: tests/test_data/model_eval/generate_synthetic_dataset_for_eval.py [hunk 4]
```diff
@@ -1,12 +1,26 @@
...
 import datetime as dt
+from pathlib import Path
 from typing import Any, Optional
 
 import numpy as np
 import pandas as pd
 from wasabi import Printer
 
 
+def add_age_is_female(df: pd.DataFrame):
```
Comment (sarakolding) [inline tests/test_data/model_eval/generate_synthetic_dataset_for_eval.py:11]: ah, maybe this is where age and sex comes in

File: tests/model_evaluation/test_visualizations.py [hunk 5]
```diff
@@ -47,23 +47,6 @@
 from psycop_model_training.utils.utils import PROJECT_ROOT, positive_rate_to_pred_probs
 
 
-@pytest.fixture(scope="function")
```
Comment (sarakolding) [inline tests/model_evaluation/test_visualizations.py:50]: where is the synth data loaded?

File: tests/conftest.py (hunk 2 above)
Comment (MartinBernstorff) [reply tests/conftest.py:25]: Just an arbitrary cut-off – faster to do than a randint :-)

File: tests/conftest.py (hunk 1 above)
Comment (MartinBernstorff) [reply tests/conftest.py:24]: Sure, modifiied

**Pull request merged**